content/en/tutorials/upgrade_gke.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "GKE Upgrades"
+linkTitle: "gkeupgrades"
+date: 2022-02-01
+description: >
+  Dealing with GKE upgrades
+---
+
+# Upgrading Google Kubernetes Engine Version
+
+The Kubernetes Project releases minor version updates every quarter, and users are expected to update their infrastructure to at least within a few minor versions of the latest Kubernetes release. With Google GKE, Opta relies on GCP's [release channels](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels) to automatically upgrade the Kubernetes control plane and worker nodes. When Opta installs your GKE cluster, it uses the "regular" GKE channel. Currently (Feb 1, 2022), this channel will install a Kubernetes 1.21.x cluster. You can run the `kubectl version` command and look for the "Server Version" key to know what version your cluster is currently running.
+
+In the next days the GKE regular channel will start updating Kubernetes to 1.22.x; which will mean that your opta-deployed Kubernetes cluster will be automatically upgraded to a 1.22.x version. This version of Kubernetes removes several deprecated Kubernetes APIs. So there are two aspects we need to proactively manage in order to prevent downtime and application issues.
```

Comment:
I think there's an official deprecation page, right? Might be good to add a link to it